### DISCUSSION TITLE
Jupyter

### DISCUSSION THREAD

**Discussion by Hemanth0324** [DrAlzahraniProjects/csusb_fall2024_cse6550_team4#9] [Polls]
I request everyone to participate here related to jupyter

**Comment by MohammadReyazT4** [comment]
[SE TASKS.docx](https://github.com/user-attachments/files/17009504/SE.TASKS.docx)
Hemanth, this is an update for my assigned task: to do ZAP and Jupyter notebook documentation for our paper chatbot project.
